openapi.json

```diff
@@ -906,7 +910,7 @@
                   },
                   "description": {
                     "type": "string",
-                    "description": "An editorial description of the event."
+                    "description": "An editorial WYSIWYG/HTML description of the event."
```

Comment:
Please reconsider this.

As I see this this is effectively changing the API by repurposing a field `description` to contain something else - both in regard to length (if I understand where you are going this might contain more text) and format (with/without HTML).

This may effect our the integration partners. 

I would suggest that we instead add a new field for the longer text. If you want a second opinion on this I suggest taking it out with @spaceo, Arne or Fini.

Reply:
Good point - I've updated it, so the new data is in a new `body` field instead.